bin/GameIndex.dbf

```diff
@@ -124,6 +124,7 @@ Region = PAL-Unk
 Serial = PBPX-95503
 Name   = Gran Turismo 3 - A-Spec [PS2 Bundle]
 Region = NTSC-U
+MemCardFilter = PBPX-95503/SCUS-97102
```

Comment:
I kinda doubt it's necessary, but I suppose it can't hurt to have the extra entry.